packages/design-system-react-native/src/components/temp-components/Blockies/Blockies.utilities.ts

```diff
@@ -0,0 +1,627 @@
+/**
```

Comment:
There are few linting ignores in the typescript version of the blockies utilities but we are achieving the exact same rendering of the blockies graphic